*immutable.SortedMap does not implement immutable.Comparer (missing Compare method)

I'm trying to go get influxdb. I get stuck at 
```

go get github.com/influxdata/influxdb
# github.com/influxdata/flux/values
../../.gvm/pkgsets/go1.15.6/global/src/github.com/influxdata/flux/values/dict.go:339:25: cannot use immutable.NewSortedMap(dictComparer(dictType)) (type *immutable.SortedMap) as type immutable.Comparer in argument to immutable.NewSortedMapBuilder:
        *immutable.SortedMap does not implement immutable.Comparer (missing Compare method)
```

how is the build marked as passing? I'm using go 1.15.6

--- a/go.mod
+++ b/go.mod
@@ -12,7 +12,7 @@ require (
 	github.com/SAP/go-hdb v0.14.1
 	github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883
 	github.com/apache/arrow/go/arrow v0.0.0-20200923215132-ac86123a3f01
-	github.com/benbjohnson/immutable v0.2.1
+	github.com/benbjohnson/immutable v0.3.0
 	github.com/bonitoo-io/go-sql-bigquery v0.3.4-1.4.0
 	github.com/c-bata/go-prompt v0.2.2
 	github.com/cespare/xxhash v1.1.0

--- a/values/dict.go
+++ b/values/dict.go
@@ -335,11 +335,7 @@ type DictionaryBuilder struct {
 // NewDictBuilder will create a new DictionaryBuilder for the given
 // key type.
 func NewDictBuilder(dictType semantic.MonoType) DictionaryBuilder {
-	builder := immutable.NewSortedMapBuilder(
-		immutable.NewSortedMap(
-			dictComparer(dictType),
-		),
-	)
+	builder := immutable.NewSortedMapBuilder(dictComparer(dictType))
 	return DictionaryBuilder{t: dictType, b: builder}
 }
 

